src/devtools/protocol.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2024 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type * as Devtools from '@yorkie-js-sdk/src/types/devtools';
```

Comment:
For now, there is a only file in `src/types`. How about moving `src/types/devtools.ts` into this folder?

`src/types/devtools.ts` -> `src/devtools/types.ts`